src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/CreateTableIndexIntegrationTest.java

```diff
@@ -405,6 +405,89 @@ public void createVectorIndexWithCorrectIndexType() {
   @Nested
   @Order(3)
   class CreateIndexFailure {
+    @Test
+    public void createIndexWithEmptydName() {
+
+      assertTableCommand(keyspaceName, testTableName)
+          .postCreateIndex(
+              """
+                      {
+                        "name": "",
+                        "definition": {
+                          "column": "vehicle_id_7"
+                        }
+                      }
+                      """)
+          .hasSingleApiError(
+              SchemaException.Code.UNSUPPORTED_SCHEMA_NAME,
+              SchemaException.class,
+              "The command attempted to create a index with a name that is not supported.",
+              "The supported index names must not be empty, more than 48 characters long, or contain non-alphanumeric-underscore characters.",
+              "The command used the unsupported index name: .");
```

Comment:
I think we need to quote schema names: this message looks very confusing as-is.

If it read:

    The command used the unsupported index name: ''.

it would be little less odd.